localisation/00_wc_artifact_localisation.csv

```diff
@@ -228,3 +228,6 @@ whispers_intrigue_1;Murmurs of the Deep Below;Murmures des profondeurs;Raunen au
 whispers_intrigue_1_desc;When someone holds this book, dark whispers creep into the corner of their mind...;Quand quelqu'un tient ce livre, de sombres murmures se glissent dans le coin de son esprit...;Wenn jemand dieses Buch in der Hand h�lt, schleicht sich ein dunkles Fl�stern in die Winkel [From.GetHerHis]es Geistes...;;;;;;;;;;;x
 whispers_learning_1;Augurs of Eldritch Insights;Pr�sages indicibles;Unheimliche Einblicke;;;;;;;;;;;x
 whispers_learning_1_desc;When someone holds this book, dark whispers creep into the corner of their mind...;Quand quelqu'un tient ce livre, de sombres murmures se glissent dans le coin de son esprit...;Wenn jemand dieses Buch in der Hand h�lt, schleicht sich ein dunkles Fl�stern in die Winkel [From.GetHerHis]es Geistes...;;;;;;;;;;;x
+# Any #;;;;;;;;;;;;;;
+heart_of_yshaarj;Heart of Y'Shaarj;;;;;;;;;;;;;x
+heart_of_yshaarj_desc;A diseased, icy mass of flesh seething with Void energies, is the actual heart of the deceased Old God, Y'Shaarj. When Y'Shaarj was torn apart by the titan Aman'Thul, its monstrous heart was the largest part of the Old God that remained intact.;;;;;;;;;;;;;x
```

Comment:
Why do you make an "any" category? If it's tier 5, it's legendary, not any

Reply:
Adding at the end of the file.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/wiki/Guidelines#localization-guidelines

Reply:
So while the whole file is sorted by tier from the beginning, you want to add new lines at the end at all cost?

Reply:
> So while the whole file is sorted by tier from the beginning, you want to add new lines at the end at all cost?

Well, you know my point - I'm for not-moving lines (thus I'm fine with the current state) and adding them at the end (thus it's easier to handle different localization of the mod).